styles/components/homepage/workflow-graphic.scss

```diff
@@ -0,0 +1,77 @@
+
+.c-workflow-graphic {
+  width: 100%;
+  margin: 40px 0;
+  padding: 64px 0 56px;
+
+  > .supply-chain-list {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: space-between;
+    height: 69px;
+
+
+    &::before {
+      display: inline-block;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      width: 710px;
+
+      transform: translate(-50%, -50%);
+      border-top: 2px solid $medium-pink;
+
+      content: '';
+      z-index: 0;
+    }
+
+    > .supply-chain-item {
+      display: flex;
+      position: relative;
+      flex-basis: 20%;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+
+      z-index: 1;
+
+      &:first-child > .icon-supply,
+      &:nth-child(4) > .icon-supply,
+      &:last-child > .icon-supply {
```

Comment:
Please use the more stable `icon-supply_chain_4` class names instead, ie if we change order or add items, it will not break the CSS.